src/main/java/service/impl/ProductServiceImp.java

```diff
@@ -0,0 +1,21 @@
+package service.impl;
+
+import model.Product;
+import service.ProductService;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ProductServiceImp implements ProductService {
+    private final List<Product> productList = new ArrayList<>();
```

Comment:
better move this list in separate class which will be prototype of database with name "Storage"